README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://img.shields.io/gitlab/pipeline/zach_nation/coremltools/master)](#)
+[![Build Status](https://travis-ci.com/apple/coremltools.svg?branch=master)](#)
```

Comment:
I think [![Build Status](https://img.shields.io/gitlab/pipeline/zach_nation/coremltools/master)](#) is correct one.

Reply:
done